read.go

```diff
@@ -30,6 +30,7 @@ var (
 	// dataset returned is still valid.
 	ErrorUnsupportedBitsAllocated = errors.New("unsupported BitsAllocated")
 	errorUnableToParseFloat       = errors.New("unable to parse float type")
+	ErrorExpectedEvenLength    = errors.New("field length is not even, in violation of DICOM spec")
```

Comment:
nit: Not sure if `gofmt` was run on this, as I think the equals sign should line up with the lines above, probably my fault because the suggestion commit feature doesn't easily take that into account 😅--if you could line it up that'd be great!